runtime/shibuya/src/lib.rs

```diff
@@ -73,7 +73,14 @@ pub const HOURS: BlockNumber = MINUTES * 60;
 pub const DAYS: BlockNumber = HOURS * 24;
 
 /// Id used for identifying assets.
+///
+/// AssetId allocation:
+/// [1; 2^32-1]     Custom user assets (permissionless)
+/// [2^32; 2^64-1]  Statemine assets (simple map)
+/// [2^64; 2^128-1] Ecosystem assets
```

Comment:
Are these ranges?
If yes, could you perhaps change it to something like `[1, 2^32-1]`?

It's a minor comment, but the Rust array init syntax had me confused.

Reply:
Yes, its ranges, symbol `,` in some notations used as decimal point separator, so, I though math notation will confuse less, but

Reply:
Ok, if you prefer the current way. Perhaps also to consider `[1 - 2^32-1]`